Specs/DateTime/Hindi/TimePeriodParser.json

```diff
@@ -0,0 +1,1327 @@
+[
+  {
+    "Input": "午後5時から6時まで不在にします。",
```

Comment:
Can you remove these Japanese specs?

Reply:
BTW, why was Japanese used as source? English should be considered the master version for localization.

Reply:
sorry, did not notice this. Not sure why the initial spec was Japanese, will check